config/prow/plugins.yaml

```diff
@@ -285,67 +285,142 @@ lgtm:
   store_tree_hash: true
 
 blockades:
+  - repos:
+    - kubernetes/api
+    - kubernetes/apiextensions-apiserver
+    - kubernetes/apimachinery
+    - kubernetes/apiserver
+    - kubernetes/cli-runtime
+    - kubernetes/cloud-provider
+    - kubernetes/cluster-bootstrap
+    - kubernetes/code-generator
+    - kubernetes/component-base
+    - kubernetes/component-helpers
+    - kubernetes/controller-manager
+    - kubernetes/cri-api
+    - kubernetes/cri-client
+    - kubernetes/csi-translation-lib
+    - kubernetes/dynamic-resource-allocation
+    - kubernetes/endpointslice
+    - kubernetes/externaljwt
+    - kubernetes/kms
+    - kubernetes/kube-aggregator
+    - kubernetes/kube-controller-manager
+    - kubernetes/kube-proxy
+    - kubernetes/kube-scheduler
+    - kubernetes/kubectl
+    - kubernetes/kubelet
+    - kubernetes/metrics
+    - kubernetes/mount-utils
+    - kubernetes/pod-security-admission
+    - kubernetes/sample-apiserver
+    - kubernetes/sample-cli-plugin
+    - kubernetes/sample-controller
+  required_labels:
```

Comment:
this config looks wrong, as do the lines below

I generated this: https://github.com/kubernetes/kubernetes/pull/131351